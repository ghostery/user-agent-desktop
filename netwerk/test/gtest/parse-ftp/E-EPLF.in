ftp> open cr.yp.to
Connected to cr.yp.to.
220 Features: a p .
Name (cr.yp.to:cyp): 
230 Hi. No need to log in; I'm an anonymous ftp server.
Remote system type is UNIX.
Using binary mode to transfer files.
ftp> ls
200 Okay.
150 Making transfer connection...
+i0.307427,m979205905,/,	tmp
+i0.631874,m951789705,/,	1998-100
+i0.236492,m977690646,r,s1172,	mirrors.html
+i0.718174,m951789707,/,	1999-275
+i0.236496,m957763516,r,s933,	1999-275.html
+i0.236247,m957763626,r,s322,	1998-401.html
+i0.689416,m951789707,/,	1998-401
+i0.714380,m951789705,/,	1997-275
+i0.236426,m957763705,r,s277,	1998-515.html
+i0.236324,m957764203,r,s325,	2000-436.html
+i0.553153,m977554031,/,	checkpwd
+i0.73117,m1025804866,/,	bib
+i0.2012,m951789707,/,	1999-541
+i0.134494,m988426955,/,	2001-260
+i0.457327,m1019068075,/,	2002-501
+i0.11567,m991428977,/,	1995-514
+i0.716165,m951789705,/,	1999-180
+i0.236327,m991363463,r,s475,	2000-515.html
+i0.687522,m951789707,/,	1998-515
+i0.412995,m951789707,/,	2000-515
+i0.213334,m958117573,/,	zpfft
+i0.295901,m1004187614,/,	psibound
+i0.236429,m951789711,r,s398,	anonftpd.html
+i0.326793,m1014690862,/,	nistp224
+i0.587605,m997428440,/,	cdb
+i0.516884,m1020211107,/,	talks
+i0.255382,m977607275,/,	ftpparse
+i0.103809,m947569810,/,	clockspeed
+i0.236433,m951789712,r,s1350,	clockspeed.html
+i0.530322,m1013028754,/,	slashdoc
+i0.10002,m1023465640,/,	export
+i0.105740,m995334516,/,	daemontools
+i0.236425,m994972628,r,s2187,	daemontools.html
+i0.236437,m999204676,r,s175,	data.html
+i0.236435,m1009503157,r,s1901,	crypto.html
+i0.236430,m1005435923,r,s739,	arith.html
+i0.236297,m1007092089,r,s2733,	cdb.html
+i0.236475,m1005436262,r,s344,	floatasm.html
+i0.126821,m977433956,/,	djbfft
+i0.236440,m951789712,r,s1319,	djbfft.html
+i0.149876,m963163007,/,	dnscache
+i0.236441,m962665064,r,s4016,	dnscache.html
+i0.674064,m994734982,/,	docs
+i0.236442,m951789712,r,s293,	docs.html
+i0.236443,m951789712,r,s952,	dot-forward.html
+i0.76895,m947569811,/,	etc-mta
+i0.236444,m951789712,r,s3174,	etc-mta.html
+i0.236445,m951789712,r,s2990,	ezmlm.html
+i0.142082,m977433956,/,	ftp
+i0.78826,m951789709,/,	im
+i0.236446,m951789712,r,s1823,	fastforward.html
+i0.236447,m951789712,r,s1707,	ftp.html
+i0.21212,m981944230,/,	freebugtraq
+i0.117314,m1022981965,/,	hardware
+i0.236326,m991364255,r,s610,	1995-514.html
+i0.63463,m977433957,/,	hash127
+i0.236450,m958703342,r,s4835,	hash127.html
+i0.236451,m951789712,r,s1515,	im.html
+i0.82651,m1013546406,/,	immhf
+i0.236299,m988235329,r,s2902,	immhf.html
+i0.147915,m980402316,/,	lib
+i0.591514,m991784750,/,	libtai
+i0.236454,m963255256,r,s2120,	libtai.html
+i0.236427,m1006883560,r,s14460,	qmail.html
+i0.236452,m999204683,r,s2153,	mail.html
+i0.236457,m951789712,r,s108,	maildir.html
+i0.71192,m951789710,/,	maildisasters
+i0.236458,m951789712,r,s873,	maildisasters.html
+i0.236459,m951789712,r,s1688,	mess822.html
+i0.13517,m1014605247,/,	mirror
+i0.236431,m1009400552,r,s581,	ntheory.html
+i0.629891,m1023118587,/,	papers
+i0.176720,m1021999815,/,	postpropter
+i0.67332,m947569812,/,	postings
+i0.101902,m947569812,/,	primegen
+i0.236453,m999204693,r,s415,	precompiled.html
+i0.236463,m951789712,r,s1121,	primegen.html
+i0.574160,m1020562675,/,	proto
+i0.236464,m951789712,r,s452,	proto.html
+i0.113471,m994717073,/,	publicfile
+i0.236295,m1007092757,r,s5651,	publicfile.html
+i0.236455,m999204697,r,s409,	qlist.html
+i0.15552,m1017725145,/,	qmail
+i0.236292,m1012629148,r,s7549,	lists.html
+i0.236468,m951789712,r,s1166,	qmailanalog.html
+i0.236456,m999204699,r,s1434,	qmsmac.html
+i0.236470,m952925886,r,s465,	rblsmtpd.html
+i0.236471,m951789712,r,s1225,	rights.html
+i0.691333,m947569813,/,	sarcasm
+i0.236465,m1000434854,r,s1058,	unix.html
+i0.236472,m951789713,r,s1568,	serialmail.html
+i0.236473,m970514024,r,s840,	sigs.html
+i0.80743,m1021999808,/,	smtp
+i0.236300,m988300727,r,s1723,	smtp.html
+i0.693264,m977433963,/,	software
+i0.236476,m951789713,r,s3671,	softwarelaw.html
+i0.710487,m947569813,/,	sortedsums
+i0.236477,m951789713,r,s2735,	sortedsums.html
+i0.23149,m1014689198,/,	speed
+i0.236478,m962664017,r,s572,	surveydns.html
+i0.589555,m1002157171,/,	surveys
+i0.236278,m1002156396,r,s3503,	surveys.html
+i0.96135,m973835888,/,	syncookies
+i0.236479,m1012529017,r,s7918,	syncookies.html
+i0.236460,m999204703,r,s1115,	tcpcontrol.html
+i0.236461,m1016453307,r,s589,	tcpip.html
+i0.236483,m951789713,r,s906,	thoughts.html
+i0.136514,m996502689,/,	threecubes
+i0.236462,m1000430758,r,s525,	time.html
+i0.121107,m1022041265,/,	ucspi-tcp
+i0.236484,m1011915691,r,s4086,	ucspi-tcp.html
+i0.236285,m999204710,r,s732,	web.html
+i0.236323,m1025804868,/,	www
+i0.236467,m1000430768,r,s2717,	y2k.html
+i0.94216,m981624489,/,	dnsroot
+i0.236280,m988005697,r,s2745,	2001-260.html
+i0.236489,m951789713,r,s291,	zeroseek.html
+i0.90373,m951789713,/,	zmodexp
+i0.236490,m958703347,r,s1513,	zmodexp.html
+i0.65452,m965092134,/,	zmult
+i0.236491,m965093093,r,s669,	zmult.html
+i0.236432,m977552387,r,s1097,	checkpwd.html
+i0.236480,m1009824436,r,s2077,	focus.html
+i0.236493,m964048291,r,s3842,	im2000.html
+i0.5981,m1025804863,/,	slashcommand
+i0.192219,m1025804864,/,	slashpackage
+i0.236277,m995235879,r,s602,	slashpackage.html
+i0.236302,m1020384933,r,s23874,	talks.html
+i0.236495,m957763499,r,s566,	1997-275.html
+i0.236494,m957763864,r,s313,	1998-100.html
+i0.543361,m951789707,/,	2000-436
+i0.236424,m957764246,r,s660,	1997-494.html
+i0.236428,m957764441,r,s730,	1999-541.html
+i0.236497,m968828938,r,s2367,	psibound.html
+i0.236500,m961639740,r,s1866,	smallfactors.html
+i0.413016,m960514519,/,	smallfactors
+i0.75096,m962492119,/,	conferences
+i0.557013,m1022046749,/,	djbdns
+i0.236282,m995342458,r,s2712,	djbdns.html
+i0.236502,m963796539,r,s216,	zpfft.html
+i0.537820,m966797797,/,	fastnewton
+i0.236503,m966797851,r,s333,	fastnewton.html
+i0.510806,m967833231,/,	patents
+i0.236448,m977607617,r,s1748,	ftpparse.html
+i0.236281,m1005436670,r,s3261,	slash.html
+i0.236279,m979753810,r,s2663,	slashdoc.html
+i0.236283,m1016442275,r,s2344,	dnsroot.html
+i0.236284,m1009400580,r,s3238,	software.html
+i0.236287,m1000434873,r,s3481,	compatibility.html
+i0.236291,m1020223834,r,s5525,	distributors.html
+i0.236293,m983072943,r,s212,	freebugtraq.html
+i0.236294,m1017547190,r,s806,	hardware.html
+i0.236296,m986022918,r,s121,	securesoftware.html
+i0.236301,m1022035196,r,s10950,	conferences.html
+i0.378623,m1007142710,/,	2001-275
+i0.236303,m991095307,r,s459,	rwb100.html
+i0.236325,m1002087642,r,s863,	nistp224.html
+i0.236436,m996502546,r,s778,	threecubes.html
+i0.236298,m1023118336,r,s27125,	papers.html
+i0.236439,m1006311069,r,s379,	bib.html
+i0.236438,m1007142093,r,s3791,	2001-275.html
+i0.236474,m1002046139,r,s169,	dh224.html
+i0.236434,m1016258736,r,s1126,	courses.html
+i0.236466,m999469348,r,s355,	slashcommand.html
+i0.236449,m1017264022,r,s1199,	patents.html
+i0.236469,m1005247635,r,s640,	mailcopyright.html
+i0.236286,m1022974070,r,s1257,	djb.html
+i0.236481,m1010383954,r,s301,	2002-501.html
+i0.236482,m1020220905,r,s1655,	export.html
+i0.236276,m1016966459,r,s6446,	index.html
+i0.236485,m1023740653,	donations.html
+i0.236486,m1016258948,r,s570,	2005-590.html
+i0.236487,m1016258749,r,s800,	2004-494.html
+i0.236488,m1023740407,	thanks.html
+i0.236498,m1022974139,r,s4459,	positions.html
+i0.236499,m1022972131,r,s466,	contact.html
226 Success.
ftp> close
ftp> 