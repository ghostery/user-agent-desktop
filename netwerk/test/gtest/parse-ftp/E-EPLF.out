<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

01-11-2001  09:38:25       <DIR>          tmp
02-29-2000  02:01:45       <DIR>          1998-100
12-24-2000  20:44:06                 1172 mirrors.html
02-29-2000  02:01:47       <DIR>          1999-275
05-08-2000  05:25:16                  933 1999-275.html
05-08-2000  05:27:06                  322 1998-401.html
02-29-2000  02:01:47       <DIR>          1998-401
02-29-2000  02:01:45       <DIR>          1997-275
05-08-2000  05:28:25                  277 1998-515.html
05-08-2000  05:36:43                  325 2000-436.html
12-23-2000  06:47:11       <DIR>          checkpwd
07-04-2002  17:47:46       <DIR>          bib
02-29-2000  02:01:47       <DIR>          1999-541
04-28-2001  03:02:35       <DIR>          2001-260
04-17-2002  18:27:55       <DIR>          2002-501
06-01-2001  20:56:17       <DIR>          1995-514
02-29-2000  02:01:45       <DIR>          1999-180
06-01-2001  02:44:23                  475 2000-515.html
02-29-2000  02:01:47       <DIR>          1998-515
02-29-2000  02:01:47       <DIR>          2000-515
05-12-2000  07:46:13       <DIR>          zpfft
10-27-2001  13:00:14       <DIR>          psibound
02-29-2000  02:01:51                  398 anonftpd.html
02-26-2002  02:34:22       <DIR>          nistp224
08-10-2001  07:27:20       <DIR>          cdb
04-30-2002  23:58:27       <DIR>          talks
12-23-2000  21:34:35       <DIR>          ftpparse
01-11-2000  05:50:10       <DIR>          clockspeed
02-29-2000  02:01:52                 1350 clockspeed.html
02-06-2002  20:52:34       <DIR>          slashdoc
06-07-2002  16:00:40       <DIR>          export
07-17-2001  01:48:36       <DIR>          daemontools
07-12-2001  21:17:08                 2187 daemontools.html
08-30-2001  20:51:16                  175 data.html
12-28-2001  01:32:37                 1901 crypto.html
11-10-2001  23:45:23                  739 arith.html
11-30-2001  03:48:09                 2733 cdb.html
11-10-2001  23:51:02                  344 floatasm.html
12-21-2000  21:25:56       <DIR>          djbfft
02-29-2000  02:01:52                 1319 djbfft.html
07-09-2000  17:16:47       <DIR>          dnscache
07-03-2000  22:57:44                 4016 dnscache.html
07-10-2001  03:16:22       <DIR>          docs
02-29-2000  02:01:52                  293 docs.html
02-29-2000  02:01:52                  952 dot-forward.html
01-11-2000  05:50:11       <DIR>          etc-mta
02-29-2000  02:01:52                 3174 etc-mta.html
02-29-2000  02:01:52                 2990 ezmlm.html
12-21-2000  21:25:56       <DIR>          ftp
02-29-2000  02:01:49       <DIR>          im
02-29-2000  02:01:52                 1823 fastforward.html
02-29-2000  02:01:52                 1707 ftp.html
02-12-2001  02:17:10       <DIR>          freebugtraq
06-02-2002  01:39:25       <DIR>          hardware
06-01-2001  02:57:35                  610 1995-514.html
12-21-2000  21:25:57       <DIR>          hash127
05-19-2000  02:29:02                 4835 hash127.html
02-29-2000  02:01:52                 1515 im.html
02-12-2002  20:40:06       <DIR>          immhf
04-25-2001  21:48:49                 2902 immhf.html
01-25-2001  05:58:36       <DIR>          lib
06-05-2001  23:45:50       <DIR>          libtai
07-10-2000  18:54:16                 2120 libtai.html
11-27-2001  17:52:40                14460 qmail.html
08-30-2001  20:51:23                 2153 mail.html
02-29-2000  02:01:52                  108 maildir.html
02-29-2000  02:01:50       <DIR>          maildisasters
02-29-2000  02:01:52                  873 maildisasters.html
02-29-2000  02:01:52                 1688 mess822.html
02-25-2002  02:47:27       <DIR>          mirror
12-26-2001  21:02:32                  581 ntheory.html
06-03-2002  15:36:27       <DIR>          papers
05-21-2002  16:50:15       <DIR>          postpropter
01-11-2000  05:50:12       <DIR>          postings
01-11-2000  05:50:12       <DIR>          primegen
08-30-2001  20:51:33                  415 precompiled.html
02-29-2000  02:01:52                 1121 primegen.html
05-05-2002  01:37:55       <DIR>          proto
02-29-2000  02:01:52                  452 proto.html
07-09-2001  22:17:53       <DIR>          publicfile
11-30-2001  03:59:17                 5651 publicfile.html
08-30-2001  20:51:37                  409 qlist.html
04-02-2002  05:25:45       <DIR>          qmail
02-02-2002  05:52:28                 7549 lists.html
02-29-2000  02:01:52                 1166 qmailanalog.html
08-30-2001  20:51:39                 1434 qmsmac.html
03-13-2000  05:38:06                  465 rblsmtpd.html
02-29-2000  02:01:52                 1225 rights.html
01-11-2000  05:50:13       <DIR>          sarcasm
09-14-2001  02:34:14                 1058 unix.html
02-29-2000  02:01:53                 1568 serialmail.html
10-02-2000  19:13:44                  840 sigs.html
05-21-2002  16:50:08       <DIR>          smtp
04-26-2001  15:58:47                 1723 smtp.html
12-21-2000  21:26:03       <DIR>          software
02-29-2000  02:01:53                 3671 softwarelaw.html
01-11-2000  05:50:13       <DIR>          sortedsums
02-29-2000  02:01:53                 2735 sortedsums.html
02-26-2002  02:06:38       <DIR>          speed
07-03-2000  22:40:17                  572 surveydns.html
10-04-2001  00:59:31       <DIR>          surveys
10-04-2001  00:46:36                 3503 surveys.html
11-10-2000  05:58:08       <DIR>          syncookies
02-01-2002  02:03:37                 7918 syncookies.html
08-30-2001  20:51:43                 1115 tcpcontrol.html
03-18-2002  12:08:27                  589 tcpip.html
02-29-2000  02:01:53                  906 thoughts.html
07-30-2001  14:18:09       <DIR>          threecubes
09-14-2001  01:25:58                  525 time.html
05-22-2002  04:21:05       <DIR>          ucspi-tcp
01-24-2002  23:41:31                 4086 ucspi-tcp.html
08-30-2001  20:51:50                  732 web.html
07-04-2002  17:47:48       <DIR>          www
09-14-2001  01:26:08                 2717 y2k.html
02-08-2001  09:28:09       <DIR>          dnsroot
04-23-2001  06:01:37                 2745 2001-260.html
02-29-2000  02:01:53                  291 zeroseek.html
02-29-2000  02:01:53       <DIR>          zmodexp
05-19-2000  02:29:07                 1513 zmodexp.html
08-01-2000  01:08:54       <DIR>          zmult
08-01-2000  01:24:53                  669 zmult.html
12-23-2000  06:19:47                 1097 checkpwd.html
12-31-2001  18:47:16                 2077 focus.html
07-19-2000  23:11:31                 3842 im2000.html
07-04-2002  17:47:43       <DIR>          slashcommand
07-04-2002  17:47:44       <DIR>          slashpackage
07-15-2001  22:24:39                  602 slashpackage.html
05-03-2002  00:15:33                23874 talks.html
05-08-2000  05:24:59                  566 1997-275.html
05-08-2000  05:31:04                  313 1998-100.html
02-29-2000  02:01:47       <DIR>          2000-436
05-08-2000  05:37:26                  660 1997-494.html
05-08-2000  05:40:41                  730 1999-541.html
09-13-2000  07:08:58                 2367 psibound.html
06-22-2000  02:09:00                 1866 smallfactors.html
06-09-2000  01:35:19       <DIR>          smallfactors
07-01-2000  22:55:19       <DIR>          conferences
05-22-2002  05:52:29       <DIR>          djbdns
07-17-2001  04:00:58                 2712 djbdns.html
07-17-2000  01:15:39                  216 zpfft.html
08-20-2000  18:56:37       <DIR>          fastnewton
08-20-2000  18:57:31                  333 fastnewton.html
09-01-2000  18:33:51       <DIR>          patents
12-23-2000  21:40:17                 1748 ftpparse.html
11-10-2001  23:57:50                 3261 slash.html
01-17-2001  17:50:10                 2663 slashdoc.html
03-18-2002  09:04:35                 2344 dnsroot.html
12-26-2001  21:03:00                 3238 software.html
09-14-2001  02:34:33                 3481 compatibility.html
05-01-2002  03:30:34                 5525 distributors.html
02-25-2001  03:49:03                  212 freebugtraq.html
03-31-2002  03:59:50                  806 hardware.html
03-31-2001  07:15:18                  121 securesoftware.html
05-22-2002  02:39:56                10950 conferences.html
11-30-2001  17:51:50       <DIR>          2001-275
05-29-2001  00:15:07                  459 rwb100.html
10-03-2001  05:40:42                  863 nistp224.html
07-30-2001  14:15:46                  778 threecubes.html
06-03-2002  15:32:16                27125 papers.html
11-21-2001  02:51:09                  379 bib.html
11-30-2001  17:41:33                 3791 2001-275.html
10-02-2001  18:08:59                  169 dh224.html
03-16-2002  06:05:36                 1126 courses.html
09-02-2001  22:22:28                  355 slashcommand.html
03-27-2002  21:20:22                 1199 patents.html
11-08-2001  19:27:15                  640 mailcopyright.html
06-01-2002  23:27:50                 1257 djb.html
01-07-2002  06:12:34                  301 2002-501.html
05-01-2002  02:41:45                 1655 export.html
03-24-2002  10:40:59                 6446 index.html
03-16-2002  06:09:08                  570 2005-590.html
03-16-2002  06:05:49                  800 2004-494.html
06-01-2002  23:28:59                 4459 positions.html
06-01-2002  22:55:31                  466 contact.html
